src/iframeMessenger.js

```diff
@@ -154,7 +154,7 @@
          * @return {int} Height integer
          */
         function _getHeight() {
-            var height = parseInt(document.body.offsetHeight, 10);
+            var height = Math.ceil(document.body.getBoundingClientRect().height) + 1;
```

Comment:
Why +1? Perhaps we could name this number with a variable to avoid a comment.

Reply:
Unfortunately there’s no good name I can come up with for the 1 here – by testing I’ve determined that when I leave it out there’s overflow, whereas when I include it there isn’t. Adding a comment sounds good though!